src/Agent/NewRelic/Agent/Core/Logging/AuditLog.cs

```diff
@@ -45,8 +45,6 @@ public static LoggerConfiguration IncludeOnlyAuditLog(this LoggerConfiguration l
 
         public static LoggerConfiguration ExcludeAuditLog(this LoggerConfiguration loggerConfiguration)
         {
```

Comment:
It gets called in several places during logger bootstrapping to ensure that we don't write audit logs to the other Serilog sinks. Because of the sequence of sink configuration, `ExcludeAuditLog()` was getting called after `IncludeOnlyAuditLog()`.

Reply:
Oh it's the same object, I see it now, thanks.